### PR TITLE
fix(learn): update Chinese challenges (Data Visualization) visualize-data-with-a-bar-chart.md

### PR DESCRIPTION
**Pull request by pjwok**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

 @ZhichengChen @XuQuan-nikkkki

### CODE REVIEW

File: curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart.md [hunk 1]
```diff
@@ -48,5 +48,10 @@ forumTopicId: 301464
 # --hints--
 
 
+# --seed--
+
```
Comment (RandellDawson) [inline curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart.md:52]: Please remove these added lines.
```suggestion
```

File: curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart.md [hunk 2]
```diff
@@ -48,5 +48,10 @@ forumTopicId: 301464
...
 # --solutions--
 
+```js
+// 请给出解决方案
+```
```
Comment (RandellDawson) [inline curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart.md:57]: Please remove these lines:

```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Please see my suggested changes.

File: curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart.md (hunk 2 above)
Comment (pjwok) [reply curriculum/challenges/chinese/04-data-visualization/data-visualization-projects/visualize-data-with-a-bar-chart.md:57]: thanks, I found that the Chinese translation of this section is the same as the original,  there is no need to change it.

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

Since none of these changes were actually needed, I am going to just close this PR.

Thank you and happy coding.

**Pull request closed without merging**